sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Features Added
 
 - Added `subscription` parameter to `AzureCliCredential` to specify the subscription to use when authenticating with the Azure CLI. ([#37994](https://github.com/Azure/azure-sdk-for-python/pull/37994))
+- Added support for environment variable `AZURE_DEFAULT_CREDENTIAL_ALLOW_LIST`.
```

Comment:
Should also mention the introduction of the `default_credential_allow_list` keyword argument in DefaultAzureCredential.